Installation/cmake/modules/CGAL_add_test.cmake

```diff
@@ -295,6 +295,7 @@ function(cgal_add_test exe_name)
       endif()
       if(EXISTS ${cmd_file})
         file(STRINGS "${cmd_file}" CMD_LINES)
+        string(CONFIGURE "${CMD_LINES}" CMD_LINES)
```

Comment:
@gdamiand prior to here is where the cmake  variable `CGAL_DATA_DIR` must be defined. From what I remember, the config file was not included before. You can start by rechecking this.

Reply:
@lrineau You suggested to define CGAL_DATA_DIR only in CGALConfig.cmake to avoid duplicated; do you see any solution to this problem please?

Reply:
Put the code in a cmake macro and use the macro wherever we need?

Reply:
@sloriot I tested, adding a message in CGAL_add_test.cmake; CGAL_DATA_DIR is defined. 
How can I test one test that uses a cmd file please ?

Reply:
you can merge in a detach or temporary branch the other PR and try to run ctest on `Surface_mesh_simplification` for example.

```
cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_tests=ON -DBUILD_TESTING=ON $CGAL_DIR
cd test/Surface_mesh_simplification
ctest -j5
```

Reply:
Thanks!

Reply:
To simplify tests; I updated one test in https://github.com/CGAL/cgal/pull/4229/commits/1c502540f52a4c2b7dec21468dfa6686d59051a9 that uses a .cmd.

I tested locally, it works.